blockchain/message_pool/src/config.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+#[allow(unused_imports)]
+use crate::MessagePool;
```

Comment:
why do we need an unused import?

Reply:
Good catch. I totally missed that. We shouldn't import `MessagePool` here. Instead, the documentation should refer to `crate::MessagePool`.

Reply:
Fixed this